Feature 1 / Log In and Sign Up design

HugoCamposLopez opened the pull request:
# Description
The design of Log In and Sign Up screens are finished.
 
 # Testing
We have tested the checkbox feature.
 
  
  ## Screenshots 
If applicable, include screenshots of the results or screenshots that help to see changes
![image](https://user-images.githubusercontent.com/93162852/146622567-6c52fac5-ecd2-4173-92d8-4507b6db8b60.png)
![image](https://user-images.githubusercontent.com/93162852/146622582-df97883e-9d45-46f3-b1e6-9d17eb156fb8.png)

HugoCamposLopez committed:
Log In Screen was updated, codeclimate added

Co-authored-by: GerardoIbarra <libarra2@ucol.mx>
Co-authored-by: OsvaldoMDLM <osvaldo.benjamin31@gmail.com>

OsvaldoMDLM committed:
LogIn screen  done

HugoCamposLopez committed:
Sign In and Log In screen updated.

The Sign In and Log In screen is finished (design)

Co-authored-by: GerardoIbarra <libarra2@ucol.mx>
Co-authored-by: OsvaldoMDLM <osvaldo.benjamin31@gmail.com>

Namoku commented:
Following [this](https://codeclimate.com/github/BC-MAY-21-RN/kata-05-reservacion-de-vuelos-k5-n21-t5/pull/7), you can refactor the parts easily, for example:
```js
<FormInput
  style={Styles.textInput}
  labelVal={password}
  secureTextEntry={true}
  onChangeText={(userpass) => setPassword(userpass)}
```
into a new component it can returns this:
```js
const ExampleNewComponent = () => {
return (
<FormInput
  style={Styles.textInput}
  labelVal={password}
  secureTextEntry={true}
  onChangeText={(userpass) => setPassword(userpass)}
....etc....
```
and you'll solve a lot of issues those you have in the code review.

YilKanda commented on ios/kata05.xcodeproj/project.pbxproj:
```diff
@@ -12,6 +12,22 @@
 		13B07FBF1A68108700A75B9A /* Images.xcassets in Resources */ = {isa = PBXBuildFile; fileRef = 13B07FB51A68108700A75B9A /* Images.xcassets */; };
 		13B07FC11A68108700A75B9A /* main.m in Sources */ = {isa = PBXBuildFile; fileRef = 13B07FB71A68108700A75B9A /* main.m */; };
 		81AB9BB82411601600AC10FF /* LaunchScreen.storyboard in Resources */ = {isa = PBXBuildFile; fileRef = 81AB9BB72411601600AC10FF /* LaunchScreen.storyboard */; };
+		636F91E599DD435689DB60B8 /* AntDesign.ttf in Resources */ = {isa = PBXBuildFile; fileRef = 8714F0FC127B406E93315F1E /* AntDesign.ttf */; };
+		E5D07E27A4C042D4A3E1BF42 /* Entypo.ttf in Resources */ = {isa = PBXBuildFile; fileRef = C02301842EEC4CBF8F299E27 /* Entypo.ttf */; };
+		DAE171A8D2BB4AB5AC32244C /* EvilIcons.ttf in Resources */ = {isa = PBXBuildFile; fileRef = 27F26C522B8B493C836A0FC9 /* EvilIcons.ttf */; };
+		F71DCB55B33241279ABB97C1 /* Feather.ttf in Resources */ = {isa = PBXBuildFile; fileRef = 1D946A442B1D44A4A19BBBFE /* Feather.ttf */; };
+		21E5DDD8B10641E8BDFC954D /* FontAwesome.ttf in Resources */ = {isa = PBXBuildFile; fileRef = 2015765BF5E546848F2220EC /* FontAwesome.ttf */; };
+		067963AEE8D3420C84CE189F /* FontAwesome5_Brands.ttf in Resources */ = {isa = PBXBuildFile; fileRef = 9C0644D06F994B95834275C8 /* FontAwesome5_Brands.ttf */; };
+		3BD9A96BEDCE456AA276BF92 /* FontAwesome5_Regular.ttf in Resources */ = {isa = PBXBuildFile; fileRef = 38B206D0ACE54C6D88541FAC /* FontAwesome5_Regular.ttf */; };
+		AAEDBE229FD94E719A1BF1AD /* FontAwesome5_Solid.ttf in Resources */ = {isa = PBXBuildFile; fileRef = 00839EF74E6A4650854EA3FA /* FontAwesome5_Solid.ttf */; };
+		66E5F665CF774CCC9BCBE10D /* Fontisto.ttf in Resources */ = {isa = PBXBuildFile; fileRef = 1CC8CC5CE38443CC99419D88 /* Fontisto.ttf */; };
+		7D128670442240F3AA807B3E /* Foundation.ttf in Resources */ = {isa = PBXBuildFile; fileRef = 153B32382F2E4463A38984CD /* Foundation.ttf */; };
+		942A027894584394847C95B4 /* Ionicons.ttf in Resources */ = {isa = PBXBuildFile; fileRef = E2BB729D95DB40CE8DB98C04 /* Ionicons.ttf */; };
+		141EB20F87764FFDB9321C87 /* MaterialCommunityIcons.ttf in Resources */ = {isa = PBXBuildFile; fileRef = D8E8D2B0A9D24A7AA67B7BA2 /* MaterialCommunityIcons.ttf */; };
+		112765ACA5AF44B9A1E14175 /* MaterialIcons.ttf in Resources */ = {isa = PBXBuildFile; fileRef = C94AB4ACBF324DCCA4955E87 /* MaterialIcons.ttf */; };
+		228A2758F3D849D4928ADC29 /* Octicons.ttf in Resources */ = {isa = PBXBuildFile; fileRef = C4FF0E88E3AC4E32B9D87AA7 /* Octicons.ttf */; };
+		8DCCA2A7F2A64F0CBA4BAF32 /* SimpleLineIcons.ttf in Resources */ = {isa = PBXBuildFile; fileRef = C542EAADECAB4F5F98AC3612 /* SimpleLineIcons.ttf */; };
+		C8FDF27E203A48E5837F185C /* Zocial.ttf in Resources */ = {isa = PBXBuildFile; fileRef = CA42A6B2B63F44B5B483936A /* Zocial.ttf */; };
```
Why are you adding multiple fonts?

HugoCamposLopez replied:
We didn't notice that. 
We installed Vector Icons and it should be added automatically.

YilKanda reviewed (changes requested):
My observation in this PR is that you are installing too many typefaces. It is advisable to have a guideline of typefaces to use. Example, I leave you the link of [FontJoy](https://fontjoy.com/), where you can make a combination of 3 fonts. But remember that the combination of many typographic fonts only incites confusion and is not intuitive for Internet users, in addition the use of a certain font is also essential since each one has a personality and transmits a message.

I imagine all these fonts are added by icons, but do you really take up all the icons from those fonts?

Commit:
Remove some lines on Checkbox

We remove some lines, that wasn't necessary

Co-authored-by: Osvaldo Medina <OsvaldoMDLM@users.noreply.github.com>

OsvaldoMDLM committed:
checkbox

Commit:
FormEmail and FormPassword created.

We create two new components and refactor the login and register screen.

Co-authored-by: Osvaldo Medina <OsvaldoMDLM@users.noreply.github.com>

Commit:
Merge branch 'feature-1' of https://github.com/BC-MAY-21-RN/kata-05-reservacion-de-vuelos-k5-n21-t5 into feature-1

Commit:
Refactored Inputs and Footer

Co-authored-by: GerardoIbarra <libarra2@ucol.mx>
Co-authored-by: OsvaldoMDLM <osvaldo.benjamin31@gmail.com>

HugoCamposLopez commented:
> Following [this](https://codeclimate.com/github/BC-MAY-21-RN/kata-05-reservacion-de-vuelos-k5-n21-t5/pull/7), you can refactor the parts easily, for example:
> 
> ```js
> <FormInput
>   style={Styles.textInput}
>   labelVal={password}
>   secureTextEntry={true}
>   onChangeText={(userpass) => setPassword(userpass)}
> ```
> 
> into a new component it can returns this:
> 
> ```js
> const ExampleNewComponent = () => {
> return (
> <FormInput
>   style={Styles.textInput}
>   labelVal={password}
>   secureTextEntry={true}
>   onChangeText={(userpass) => setPassword(userpass)}
> ....etc....
> ```
> 
> and you'll solve a lot of issues those you have in the code review.

Thank you, we just did that, and now we have less code lines.

YilKanda commented on src/components/checkBox/CheckBox.js:
```diff
@@ -0,0 +1,39 @@
+    </View>
+  )
+}
+
+
+
+
+
+
+
```
Eliminate unnecessary spaces.

YilKanda commented on src/components/checkBox/CheckBoxStyle.js:
```diff
@@ -0,0 +1,19 @@
+    }
+})
+
+export default CheckBoxStyle
```
Add a blank line.

YilKanda commented on src/components/googleButton/GoogleButton.js:
```diff
@@ -0,0 +1,21 @@
+
+    </TouchableOpacity>
+  )
+}
```
Add a blank line.

YilKanda commented on src/components/googleButton/GoogleButtonStyle.js:
```diff
@@ -0,0 +1,29 @@
+    width: 30,
+    height: 30,
+  }
+});
```
Add a blank line.

YilKanda commented on src/components/index.js:
```diff
@@ -1 +1,7 @@
+export {GoogleButton} from './googleButton/GoogleButton'
+export {InputPassword} from './Forminput/InputPassword'
+export {InputEmail} from './Forminput/InputEmail'
+export {Footer} from './Footer/Footer'
```
Add a blank line.

YilKanda commented on src/helpers/size.js:
```diff
@@ -0,0 +1,3 @@
+import {Dimensions} from 'react-native';
+export const windowWith = Dimensions.get('window').width;
+export const windowHeight = Dimensions.get('window').height;
```
Add a blank line.

YilKanda commented on src/screens/login.js:
```diff
@@ -1,27 +1,44 @@
 export const login = (props) => {
+  const {navigation} = props
+  return (
+
```
Remove the unnecessary spaces.

HugoCamposLopez replied:
We have done what you asked and we deleted the fonts that we are not using.

YilKanda reviewed (commented):
Before uploading your *.js files, I recommend you make use of [ESLint](https://eslint.org/docs/user-guide/getting-started), to verify the formatting of your code and, thus, keep it as readable as possible.

There are many unnecessary spaces in your files.

Also, remember to put a blank line at the end of each file.

Commit:
Fonts, Ident and components updated.

The fonts were delete it.
Ident is good and checked with Eslint.
Components are separate and we have less code lines in register and log in screens.

Co-authored-by: Osvaldo Medina <OsvaldoMDLM@users.noreply.github.com>
Co-authored-by: Luis Ibarra <GerardoIbarra@users.noreply.github.com>

YilKanda reviewed (commented):
Check the indentation of your files.
